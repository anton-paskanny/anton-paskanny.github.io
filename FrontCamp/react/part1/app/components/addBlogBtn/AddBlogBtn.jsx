import React from 'react';

import './styles.css';

export default (props) => (
  <button className="add-blog-btn" onClick={props.togglePopup}>&#43;</button>
);
