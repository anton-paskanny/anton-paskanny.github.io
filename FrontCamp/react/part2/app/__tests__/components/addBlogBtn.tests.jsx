import React from 'react';
import ReactDOM from 'react-dom';
import { shallow, configure, mount } from 'enzyme';
import renderer from 'react-test-renderer';
import Adapter from 'enzyme-adapter-react-16';

import AddBlogBtn from '../../client/components/addBlogBtn/AddBlogBtn';

configure({ adapter: new Adapter() });

describe('AddBlogBtn component', () => {
  test('should render AddBlogBtn component', () => {
    const component = shallow(<AddBlogBtn />);
    expect(component).toHaveLength(1);
  });

  test('should render correctly', () => {
    const component = renderer.create(<AddBlogBtn />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
