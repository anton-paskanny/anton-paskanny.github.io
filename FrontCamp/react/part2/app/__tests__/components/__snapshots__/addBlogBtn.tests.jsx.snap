// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddBlogBtn component should render correctly 1`] = `
<button
  className="add-blog-btn hoverable"
  onClick={undefined}
>
  +
</button>
`;
